Sources/SwiftGRPCNIO/GRPCClientChannelHandler.swift

```diff
@@ -192,8 +208,6 @@ extension GRPCClientChannelHandler: ChannelOutboundHandler {
 extension GRPCClientChannelHandler {
   /// Closes the HTTP/2 stream. Inbound and outbound state are set to ignore.
   public func close(context: ChannelHandlerContext, mode: CloseMode, promise: EventLoopPromise<Void>?) {
-    self.observeError(GRPCError.client(.cancelledByClient))
```

Comment:
Should we send a user event here instead?

Reply:
No; the previous mechanism for cancelling was sending a `close` which we'd pick up here which was a bit of a hack.

Reply:
Duh, right. I missed that above. Sorry!